src/UnitTests/GitHub.VisualStudio/UI/Views/GitHubPaneViewModelTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using GitHub.Api;
+using GitHub.Exports;
+using GitHub.Models;
+using GitHub.Primitives;
+using GitHub.Services;
+using GitHub.UI;
+using GitHub.VisualStudio;
+using GitHub.VisualStudio.UI.Views;
+using NSubstitute;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Reactive.Linq;
+using UnitTests;
+using Xunit;
+
+public class GitHubPaneViewModelTests : TestBaseClass
+{
+    private readonly IServiceProvider serviceProvider;
+    private readonly IUIController uiController;
+    private readonly FakeMenuCommandService menuCommandService;
+    private readonly GitHubPaneViewModel viewModel;
+    private UIViewType lastUiControllerJump;
+
+    public GitHubPaneViewModelTests()
+    {
+        #region Start tests already logged in with an active repo.
+
+        var repositoryHosts = Substitutes.RepositoryHosts;
+        repositoryHosts.IsLoggedInToAnyHost.Returns(true);
+
+        var teamExplorerServiceHolder = Substitute.For<ITeamExplorerServiceHolder>();
+        var activeRepo = Substitute.For<ILocalRepositoryModel>();
+        activeRepo.CloneUrl.Returns(new UriString("https://github.com/foo/foo"));
+        teamExplorerServiceHolder
+            .When(_ => _.Subscribe(Arg.Any<object>(), Arg.Any<Action<ILocalRepositoryModel>>()))
```

Comment:
The `_` symbol as a lamda parameter usually means ["this parameter is ignored"](http://stackoverflow.com/questions/2778362/what-does-this-mean-in-lambda-expressions), but here you use it. `x` is our usual first choice for simple lambda params.